README.md

```diff
@@ -68,16 +68,16 @@ Where `.request-log.js` pulls all the logs that have `req` tags, `.error-log.js`
 
 ### File rotation _only_
 
-To disable formatting logs into JSON and take advantage of file rotation only, simply add `isJson: false` to the config file options:
+To disable formatting logs into JSON and take advantage of file rotation only, simply add `isJson: false` to the `output` object in your configuration file:
 
 ```javascript
 module.exports = {
   filter(data) {return !!data.req},
   output: {
     path: "request.log",
+    isJson: false // JSON formatting will be disabled,
     options: {
```

Comment:
can you annotate that these are options passed to `rotating-file-stream`? simply because this example is for using that specifically